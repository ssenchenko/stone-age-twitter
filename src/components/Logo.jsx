import React from 'react';
import styled from 'styled-components';

const LogoStyled = styled.h1`
  color: darkgreen;
  font-weight: bold;
  font-size: 32px;
`;

const Logo = () => (
  <LogoStyled>Test</LogoStyled>
);

export default Logo;
