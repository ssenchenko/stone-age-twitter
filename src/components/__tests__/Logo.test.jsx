import 'jest-styled-components';
import React from 'react';
import renderer from 'react-test-renderer';
import { ThemeProvider } from 'styled-components';

import DefaultTheme from '../../settings/themes';
import Logo from '../Logo';

describe('Logo', () => {
  it('renders correctly', () => {
    const tree = renderer
      .create(
        <ThemeProvider theme={DefaultTheme}>
          <Logo />
        </ThemeProvider>,
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
