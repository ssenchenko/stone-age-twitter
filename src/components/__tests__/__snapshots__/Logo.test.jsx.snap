// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Logo renders correctly 1`] = `
.c0 {
  color: darkgreen;
  font-weight: bold;
  font-size: 32px;
}

<h1
  className="c0"
>
  Test
</h1>
`;
